src/components/screens/ScoreSetCreator.vue

```diff
@@ -0,0 +1,2021 @@
+                        <template #content>
+                          <h3 class="target-header">External Identifier Data</h3>
+                          <div v-for="externalId of slotProps.data.externalIdentifiers" class="compact-target"
+                            :key="externalId">
+                            <p class="compact-target">
+                              <strong>{{ externalId.identifier.dbName }}:</strong> {{ externalId.identifier.identifier
+                              }}, <strong>Offset:</strong> {{ externalId.offset }}
+                            </p>
+                            <span
+                              v-if="validationErrors[`targetGenes.${slotProps.data.index}.externalIdentifiers.${externalId.identifier.dbName}.identifier`]"
+                              class="mave-field-error">{{
+                                validationErrors[`targetGenes.${slotProps.data.index}.externalIdentifiers.${externalId.identifier.dbName}.identifier`]
+                              }}</span><br>
+                            <span
+                              v-if="validationErrors[`targetGenes.${slotProps.data.index}.externalIdentifiers.${externalId.identifier.dbName}.offset`]"
+                              class="mave-field-error">{{
+                                validationErrors[`targetGenes.${slotProps.data.index}.externalIdentifiers.${externalId.identifier.dbName}.offset`]
+                              }}</span>
+                          </div>
+                        </template>
+                      </Card>
+                    </template>
+                    <template #footer>
+                      <div class="flex flex-wrap justify-content-start gap-2">
+                        <Button icon="pi pi-times-circle" label="Clear all" severity="secondary" class="p-button-help"
+                          @click="targetsCleared" />
+                      </div>
+                    </template>
+                  </DataTable>
+                </span>
+              </div>
+              <span v-if="validationErrors['targetGenes']" class="mave-field-error">{{validationErrors['targetGenes']}}</span>
+              <div class="field">
+                <Message v-if="targetGenes?.length > 1" severity="info">
+                  When defining variants against multiple targets, uploaded variant coordinates should be fully
+                  qualified with respect to target names or target accessions.
+                </Message>
+              </div>
+              <div class="mavedb-wizard-step-controls-row">
+                <div class="flex justify-content-between mavedb-wizard-step-controls pt-4">
+                  <Button label="Back" severity="secondary" icon="pi pi-arrow-left" @click="showPreviousWizardStep" />
+                  <Button label="Next" :disabled="this.maxWizardStepValidated < activeWizardStep" icon="pi pi-arrow-right" iconPos="right" @click="showNextWizardStepIfValid(showNextWizardStep)" />
```

Comment:
This step is automatically valid, but we should probably enforce that the user has added at least one target.

Reply:
done by your suggested change